build.assets/version-ops/cmd/query-latest/main_test.go

```diff
@@ -0,0 +1,110 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package main
```

Comment:
Add an extra newline between license and package clause, otherwise `go doc` will render the license as documentation.

Reply:
Done in affbe4648a7cc9511ff11ce690c161333d71461d, b40925a19dfaa2f00050d3258b9fbc7333b9f686, c32db91716fc6e9bbe06a4efad0aeaa460105bda.